src/benchmark_llm_serving/bench_suite.py

```diff
@@ -217,10 +224,26 @@ def main():
 
     now = utils.get_now()
     logger.info(f"{now} Drawing graphs")
-    draw_and_save_graphs(output_folder, speed_threshold=args.speed_threshold)
+    draw_and_save_graphs(output_folder, speed_threshold=args.speed_threshold, gpu_name=args.gpu_name,
+                        min_number_of_valid_queries=args.min_number_of_valid_queries)
     now = utils.get_now()
     logger.info(f"{now} Drawing graphs : DONE")
 
+    now = utils.get_now()
+    logger.info(f"{now} Making readme")
+    make_readme(output_folder)
+    now = utils.get_now()
+    logger.info(f"{now} Making readme : DONE")
+
+    now = utils.get_now()
+    logger.info(f"{now} Zipping raw_results folder")
+    shutil.make_archive(os.path.join(output_folder, "raw_results"), 'zip', raw_results_folder)
+    shutil.rmtree(raw_results_folder)
```

Comment:
Why a rmtree method is applied on raw_results_folder?

Reply:
Because we zipped the folder with the line above so we delete it in order to take less disk space and be able to push it to github "as is".